lib/cache/cache.go

```diff
@@ -1039,6 +1047,16 @@ func (c *Cache) GetClusterConfig(opts ...services.MarshalOption) (types.ClusterC
 	return rg.clusterConfig.GetClusterConfig(opts...)
 }
 
+// GetClusterAuditConfig gets ClusterAuditConfig from the backend.
+func (c *Cache) GetClusterAuditConfig(ctx context.Context, opts ...services.MarshalOption) (types.ClusterAuditConfig, error) {
```

Comment:
IIUC you need to add the following to `lib/auth/auth.go` for the cache to be used by the auth server.
```go
// GetClusterAuditConfig gets ClusterAuditConfig from the backend.
func (a *Server) GetClusterAuditConfig(ctx context.Context, opts ...services.MarshalOption) (types.ClusterAuditConfig, error) {
	return a.GetCache().GetClusterAuditConfig(ctx)
}
```

Reply:
I don't think so. IIUC the custom getters in `lib/auth/auth.go` (overriding the getters that would otherwise be provided by the [embedded `Services`](https://github.com/gravitational/teleport/blob/6eccd829bd285d2c32f8311bbddc2ee61e05b3a5/lib/auth/auth.go#L262-L264)) are just helpers for when you have `auth.Server` directly at disposal. That's usually not the case though, and what's available is a caching access point instead.

Reply:
It occurred to me this might speed up auth server startup a little so I'll actually add the overrides as you'd suggested.